web/src/components/nomination/NominateForm.vue

```diff
@@ -337,12 +337,10 @@ export default {
       this.clearAlert();
 
       const payload = this.parseFormData();
-      if (process.env.NODE_ENV === 'production') {
-        this.$db('People').create(payload, { typecast: true }, this.afterSave);
-      } else {
+      if (process.env.NODE_ENV === 'development') {
         console.log(payload);
-        this.setAlert('success', this.$t('nominateSpeaker.thanks'));
       }
+      this.$db('People').create(payload, { typecast: true }, this.afterSave);
```

Comment:
Perhaps we should still protect this path under 'production', as a new user that doesn't read the instructions might not set NODE_ENV to development. Though I suppose without the Airtable secret, their actions might be limited.

Reply:
Oh, they don't need to set it manually, `yarn serve` runs on development mode by default and if they just copy the `.env.sample` file, it also has `NODE_ENV=development` there. And you're right, without the access key to production database they can't connect at all, so there's nothing to worry here. 👍